test/unit/chips/ChipSet.test.js

```diff
@@ -46,115 +46,81 @@ test('#adapter.hasClass returns false if component does not contains class', ()
 });
 
 test('#adapter.setSelected adds selectedChipId to state', () => {
-  const wrapper = shallow(<ChipSet><div id='1' /></ChipSet>);
+  const getSelectedChipIds = td.func();
+  const handleSelect = td.func();
+  const foundation = {getSelectedChipIds};
+  const wrapper = shallow(<ChipSet handleSelect={handleSelect}><div id='1' /></ChipSet>);
+  wrapper.setState({foundation});
+  td.when(getSelectedChipIds()).thenReturn(['1']);
+
   wrapper.instance().adapter.setSelected('1', true);
-  assert.isTrue(wrapper.state().selectedChipIds.has('1'));
+  assert.isTrue(wrapper.state().selectedChipIds.indexOf('1') > -1);
+  td.verify(handleSelect(['1']), {times: 1});
+  td.verify(getSelectedChipIds(), {times: 1});
 });
 
 test('#adapter.setSelected removes selectedChipId from state', () => {
   const wrapper = shallow(<ChipSet><div id='1' /></ChipSet>);
   wrapper.setState({selectedChipIds: new Set(['1'])});
   wrapper.instance().adapter.setSelected('1', false);
-  assert.isFalse(wrapper.state().selectedChipIds.has('1'));
+  assert.isFalse(wrapper.state().selectedChipIds.indexOf('1') > -1);
 });
 
 test('#adapter.setSelected removes selectedChipId from state', () => {
```

Comment:
I think this test an exact duplicate of the one above lol

Reply:
ugh - too much copy paste :(